buildchain/buildchain/salt_tree.py

```diff
@@ -302,6 +301,8 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/addons/solutions/deployed/init.sls'),
     Path('salt/metalk8s/addons/solutions/deployed/namespace.sls'),
 
+    Path('salt/metalk8s/addons/storageclass.sls'),
```

Comment:
To be consistent I would move it to `salt/metalk8s/addons/storageclass/deployed.sls`

Reply:
done